docs/changelog.rst

```diff
@@ -10,6 +10,8 @@ This page details the changes in the various ``django-formtools`` releases.
 
 - Added Python 3.10 to test matrix.
 
+- Added support for Django 4.0.
```

Comment:
I would say "Confirmed", we didn't fix anything here:
```suggestion
- Confirmed support for Django 4.0.
```

Reply:
Yes, sounds better. Thanks again for the suggestion!